modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/js/ckeditor5/ClassicEditor.tsx

```diff
@@ -110,6 +110,7 @@ const ClassicEditor = ({
 		<div className={`lfr-ck ${className ? className : ''}`}>
 			<CKEditor
 				config={{
+					licenseKey: 'GPL',
```

Comment:
As I said, every single use of CKEditor in DXP _should_ support EditorConfigContributors. This fallback is for the case it doesn't.